app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -65,4 +68,36 @@ class ProfileChooserViewModel @Inject constructor(
 
     return sortedProfileList
   }
+
+  fun addViewVisibility(binding: ProfileChooserAddViewBinding, boolean: Boolean) {
+    when (boolean) {
+      false -> {
+        binding.profileChooserAddViewFirstTime.visibility = View.VISIBLE
+        binding.profileChooserAddView.visibility = View.GONE
+      }
+      true -> {
+        binding.profileChooserAddView.visibility = View.VISIBLE
+        binding.profileChooserAddViewFirstTime.visibility = View.GONE
+      }
+
+    }
+
+  }
+
+  fun profileViewVisibility(binding: ProfileChooserProfileViewBinding, boolean: Boolean) {
```

Comment:
Please do not pass binding in viewModel also name your variables correctly.

Reply:
Okay,PTAL :)